app/helpers/request_info_helper.rb

```diff
@@ -0,0 +1,25 @@
+module RequestInfoHelper
+  def provision_tab_configuration(workflow)
+    prov_tab_labels = []
+    workflow.provisioning_tab_list.each do |dialog|
+      prov_tab_labels.push({:name => dialog[:name], :text => dialog[:description]})
+    end
+    _prov_tab_labels, _prov_tab_keys = prov_tab_labels, request_info_types.keys
```

Comment:
These assignments are unnecessary.

```suggestion
    return prov_tab_labels, request_info_types.keys
```